docs/apis/experimental.md

```diff
@@ -0,0 +1,60 @@
+# Experimental
+
+```{eval-rst}
+.. automodule:: experimental.__init__
+   :members:
+```
+
+## Cell Space
```

Comment:
Does this whole part need to be hard coded? That means on each change we have to also modify this file right?

Reply:
If you create new scripts you need to add them here. At least that's how I've done it in the past and how the other .md files in the directory are doing it e.g. `visualisation.md`. Are you thinking sphinx might have a smarter way of doing it?

Reply:
Yeah, probably.

That can be done in another PR however, so this is good to go!